lib/ui/widgets/common/custom_drawer/custom_drawer.dart

```diff
@@ -0,0 +1,137 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:receipe_app/generated/l10n.dart';
+import 'package:receipe_app/ui/common/app_colors.dart';
+import 'package:receipe_app/ui/common/app_images.dart';
+import 'package:receipe_app/ui/common/ui_helpers.dart';
+import 'package:receipe_app/ui/extension/app_typography.dart';
+import 'package:receipe_app/ui/extension/palette.dart';
+import 'package:stacked/stacked.dart';
+
+import 'custom_drawer_model.dart';
+
+class CustomDrawer extends StackedView<CustomDrawerModel> {
+  const CustomDrawer({super.key});
+
+  @override
+  Widget builder(
+    BuildContext context,
+    CustomDrawerModel viewModel,
+    Widget? child,
+  ) {
+    ThemeData theme = Theme.of(context);
+    AppTypography? typography = theme.extension<AppTypography>();
+    Palette? palette = theme.extension<Palette>();
+    return Drawer(
+      child: Column(
```

Comment:
I don't think our drawer doesn't have a background color for you to start your work with a `Column`.